tests/test.js

```diff
@@ -144,11 +144,7 @@ describe('Mocking rejects', () => {
 
   it('mocking rejects', async () => {
     fetch.mockRejectOnce('fake error')
-    try {
-      await APIRequest2('google')
-    } catch (e) {
-      expect(e).toEqual('fake error')
-    }
+    return expect(APIRequest2('google')).rejects.toEqual('fake error')
```

Comment:
The original style was violating lint rules so I corrected it to be the more succinct style.